cdk/lib/__snapshots__/prism.test.ts.snap

```diff
@@ -642,13 +625,9 @@ dpkg -i /prism/prism.deb",
           },
         ],
         "Tags": Array [
-          Object {
```

Comment:
A few resources are losing their `App` tag in this PR - should we be concerned about that?

Reply:
Yes we should be, thanks for flagging this! It looks like more constructs need to take in the `app` prop than were updated in https://github.com/guardian/cdk/pull/326. We'll need to fix this before merging this in.

Reply:
Making this PR draft until @guardian/cdk has been updated to fix this.

Reply:
https://github.com/guardian/cdk/pull/378 should solve this.